latest/ug/ml/node-efa.adoc

```diff
@@ -124,17 +124,114 @@ If you don't have an existing cluster, you can run the following command to crea
 eksctl create cluster -f efa-cluster.yaml
 ----
 +
-NOTE: Because the instance type used in this example has GPUs, `eksctl` automatically installs the NVIDIA Kubernetes device plugin on each instance for you.
-. Deploy the EFA Kubernetes device plugin.
+NOTE: Because the instance type used in this example has GPUs, `eksctl` automatically installs the NVIDIA Kubernetes device plugin on each instance for you. When `efaEnabled: true` is set in your configuration, `eksctl` also automatically deploys the EFA device plugin on your nodes.
```

Comment:
we should add a clarifying note that the NVIDIA device plugin is built into Bottlerocket's EKS NVIDIA variant and therefore not required nor automatically deployed when using `eksctl` with Bottlerocket as the OS